pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSampleProjects.sql

```diff
@@ -42,4 +43,10 @@ LEFT JOIN (
   ) items
   GROUP BY project_projectId
 ) stats ON stats.project_projectId = proj.projectId
-LEFT JOIN Contact c ON c.contactId = proj.contactId
+LEFT JOIN Project_Contact pc ON pc.projectId = (
+  SELECT 
+    pc1.projectId
+  FROM Project_Contact pc1 WHERE proj.projectId = pc1.projectId AND pc1.contactRoleId = 1
+  ORDER BY pc1.projectId LIMIT 1
+)
```

Comment:
`LIMIT` applies to the entire query, which means this will only select 1 contact - not 1 for each project as you intended